pymc/testing.py

```diff
@@ -958,5 +959,18 @@ def seeded_numpy_distribution_builder(dist_name: str) -> Callable:
 
 
 def assert_no_rvs(var):
```

Comment:
IIRC, would the name `assert_no_measurable_rvs` be more fitting?

Reply:
See reasoning above